app/main/view.ts

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+import { BrowserView } from 'electron';
+
+import ConfigUtil = require('../renderer/js/utils/config-util');
+const shouldSilentWebview = ConfigUtil.getConfigItem('silent');
+
+class View extends BrowserView {
+	// TODO: Typescript - Create an interface for props here
```

Comment:
Since you know what the props are going to be why this `TODO`? Just create the interface here.

Reply:
Done.